src/lib/auth/actions.ts

```diff
@@ -12,7 +12,7 @@ const AUTH_REDIRECT_ORIGIN =
   env.NODE_ENV === "production" ? "https://promptracer.dev" : "http://localhost:3000"
 const AUTH_REDIRECT_URL = `${AUTH_REDIRECT_ORIGIN}/auth/callback`
 
-export const loginWithGitHubAction = action.action(async ({}) => {
+export const loginWithGitHubAction = action.action(async ({ }) => {
```

Comment:
is this your format on save?

definitely gotta figure out what's going on

Reply:
yeaah lmao I wonder if my prettier settings are scuffed

Reply:
fixed this turns out it was my default formatter got reset 💀